boards/PDM/Inc/Io/configs/Io_EfuseConfig.h

```diff
@@ -0,0 +1,463 @@
+#pragma once
```

Comment:
I'm still for getting rid of defines that we don't need for this PR.

Reply:
I think we can include a github link to a previous commit of this file as a reference if we ever need the constants in the future.

Reply:
The other efuse's haven't been properly configured, we shouldn't delete anything yet